### PR TITLE
Add function git_version build_version

### PR DESCRIPTION
**Pull request by forsaken628**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #8436

## What this PR does / why we need it:
Add function git_version build_version

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package version
+
+var (
+	// GoVersion go version, setup by makefile
+	GoVersion = ""
```
Comment (forsaken628) [inline pkg/version/version.go:5]: why no use `runtime.Version()`

**Comment by taofengliu** [comment]
Thanks for your contribution!

File: pkg/version/version.go
Comment (taofengliu) [inline pkg/version/version.go:1]: License information is needed for this file.

File: pkg/sql/plan/function/builtin/multi/build_version.go
Comment (taofengliu) [inline pkg/sql/plan/function/builtin/multi/build_version.go:1]: `select build_version();` panic.
2023-03-24 16-12-34 的屏幕截图

environment: ubuntu20.04

**Review by taofengliu — CHANGES_REQUESTED** [review]

File: pkg/version/version_test.go
Comment (taofengliu) [inline pkg/version/version_test.go:1]: No need for this test.

File: pkg/sql/plan/function/builtin/multi/build_version.go
Comment (forsaken628) [reply pkg/sql/plan/function/builtin/multi/build_version.go:1]: 似乎makefile 里 $(shell date) 并不是一个稳定的格式。。怎么办呢？

File: cmd/mo-service/config.go [hunk 2]
```diff
@@ -37,7 +36,9 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/logutil"
 	"github.com/matrixorigin/matrixone/pkg/pb/metadata"
 	"github.com/matrixorigin/matrixone/pkg/perfcounter"
+	"github.com/matrixorigin/matrixone/pkg/util/metric/stats"
 	tomlutil "github.com/matrixorigin/matrixone/pkg/util/toml"
+	_version "github.com/matrixorigin/matrixone/pkg/version"
```
Comment (zhangxu19830126) [inline cmd/mo-service/config.go:41]: Do not use underscores at the beginning of package names

**Review by zhangxu19830126 — CHANGES_REQUESTED** [review]

File: cmd/mo-service/config.go (hunk 2 above)
Comment (forsaken628) [reply cmd/mo-service/config.go]: but “version” is a conflicting ident

**Review by taofengliu — APPROVED** [review]

**Comment by taofengliu** [comment]
Sorry for delay. Thanks a lot for your contribution!

**Pull request merged**